content/blog/a-month-after-eclipsecon-adoptium-community-day-summary-and-more/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: A month after EclipseCon: Adoptium Community day summary, and more.
+date: "2022-11-24T12:00:00+00:00"
+author: carmendelgado
+description: Our Community Manager gives us a summary of Eclipse Adoptium activities around the EclipseCon event.
+tags:
+  - "Adoptium"
+  - "EclipseCon"
+---
+
+A month after the EclipseCon. I, Carmen Delgado, Adoptium Community manager, want to share my experience attending the [EclipseCon 2022](https://www.eclipsecon.org/2022) edition in Ludwigsburg, Germany, from 24 to  27 October.  It was my third week in the role, and this is how I documented the experience.
+
+## Monday, the 24th: Community Day
+
+This day I had the opportunity to meet most of my Eclipse co-workers during our breakfast at the hotel. We had a morning session where Tim Ellison, George Adams, and Shelley Lambert went through the Adoptium working group status and all the projects, and the community gave some feedback. You can read the presentation [here](Adoptium-Community-Day-EclipseCon2022.pdf).
```

Comment:
Would it be helpful to add embedded links to more information about who Tim, George, and Shelley are? Or to describe them briefly?

Reply:
Added the same title they shared during the community day, thank for the suggestion.